pkg/test/resourcefixture/testdata/basic/redis/v1beta1/redisCluster/update.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: redis.cnrm.cloud.google.com/v1
```

Comment:
The version should be  `v1alpha1`.
So we fully control the release of the ConfigConnector versions and it has to start from Alpha. Right now, v1beta1 means stable to ConfigConnector users.

Reply:
updated the rediscluster version to v1alpha1 and left the dependencies version as is considering those are already stable. Is that the correct way?